### ISSUE TITLE
Unable to update profile specialization

### ISSUE THREAD

**Issue by Zooce** [mehmetbajin/gt-course-surveys#40]
While trying to update my specialization in my profile to Computational Perception and Robotics, the  "Updated" message pops up as if everything worked correctly, but when I go back to my profile after visiting another page or logging out, the specialization is one of the other specializations, not Computational Perception and Robotics.

Also, I noticed that the first time I tried updating the specialization, I would always get some Firebase error that it found an undefined value in the specialization. To get past that Firebase error, I had to change the specialization to something else (like Machine Learning).

The bottom line: I think there's something wrong with the Computational Perception and Robotics value in Firebase, which is preventing it from being selected.

**Comment by mehmetbajin** [comment]
Great find! Classic case of `0` being interpreted as `falsy`. Fixed with latest deploy just now.
